src/Stratis.Bitcoin.Features.Consensus/StakeValidator.cs

```diff
@@ -143,26 +329,88 @@ private static uint256 ToUInt256(BigInteger input)
             return new uint256(array, false);
         }
 
+        /// <summary>
+        /// Converts <see cref="uint256" /> to <see cref="BigInteger" />.
+        /// </summary>
+        /// <param name="input"><see cref="uint256"/> input value.</param>
+        /// <returns><see cref="BigInteger"/> version of <paramref name="input"/>.</returns>
         private static BigInteger FromUInt256(uint256 input)
         {
             return BigInteger.Zero;
         }
 
-        // Stratis kernel protocol
-        // coinstake must meet hash target according to the protocol:
-        // kernel (input 0) must meet the formula
-        //     hash(stakeModifierV2 + stakingCoins.Time + prevout.Hash + prevout.N + transactionTime) < target * weight
-        // this ensures that the chance of getting a coinstake is proportional to the
-        // amount of coins one owns.
-        // The reason this hash is chosen is the following:
-        //   stakeModifierV2: Scrambles computation to make it very difficult to precompute future proof-of-stake.
-        //   stakingCoins.Time: Time of the coinstake UTXO. Slightly scrambles computation.
-        //   prevout.Hash: Hash of stakingCoins UTXO, to reduce the chance of nodes generating coinstake at the same time.
-        //   prevout.N: Output number of stakingCoins UTXO, to reduce the chance of nodes generating coinstake at the same time.
-        //   transactionTime: Timestamp of the coinstake transaction.
-        //   Block or transaction tx hash should not be used here as they can be generated in vast
-        //   quantities so as to generate blocks faster, degrading the system back into a proof-of-work situation.
-        //
+        /// <summary>
+        /// Returns <c>true</c> if provided UTXOs were confirmed in less then <see cref="maxDepth" /> number of blocks.
+        /// </summary>
+        /// <param name="utxoSet">The utxo set.</param>
+        /// <param name="indexFrom">Chained block from which we are counting the depth.</param>
+        /// <param name="targetDepth">The target depth.</param>
+        /// <returns><c>true</c> if [is confirmed in n previous blocks] [the specified utxo set]; otherwise, <c>false</c>.</returns>
```

Comment:
why those `[` and `]` ? make normal English sentence
`<c>true</c> if the coins were spent within N blocks from <see cref="indexFrom"/>, <c>false</c> otherwise.`